rosa_planning/rosa-sts-aws-prereqs.adoc

```diff
@@ -34,3 +34,6 @@ include::modules/osd-aws-privatelink-firewall-prerequisites.adoc[leveloffset=+1]
 == Additional resources
```

Comment:
Needs a unique ID, e.g. from guidelines:
```
[role="_additional-resources"]
[id="additional-resources_configuring-alert-notifications"]
== Additional resources
```
But! if you break the stuff I mentioned above into modules, it is possible all of these would be distributed throughout the assembly under the modules they go with anyway. So use your judgement here :slightly_smiling_face: